tests/e2e/cucumber/steps/app-files/link.ts

```diff
@@ -25,6 +34,16 @@ When(
   }
 )
 
+When(
+  '{string} renames the most recently created public link of space to {string}',
+  async function (this: World, stepUser: any, newName: any): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const linkObject = new objects.applicationFiles.Link({ page })
+    const linkName = await linkObject.changeName({ newName, space: true })
+    expect(newName).toBe(linkName)
```

Comment:
it's not good to add assertions inside the when step.

Reply:
we aren't using then steps unless it's absolutely necessary in e2e not our decision so assertions needs to be done on when step

https://github.com/owncloud/web/pull/7378#discussion_r936402653

Reply:
We don't use the **Then** step so assertions were done inside the **When** step  if need